src/pages/ModelDetails/ModelDetails.tsx

```diff
@@ -38,9 +38,9 @@ export default function ModelDetails() {
     async function loadFullData() {
       try {
         const response = await startModelWatcher(modelUUID, appState, dispatch);
-        conn = response?.conn ?? null;
-        watcherHandle = response?.watcherHandle ?? null;
-        pingerIntervalId = response?.pingerIntervalId ?? null;
+        conn = response.conn;
+        watcherHandle = response.watcherHandle;
+        pingerIntervalId = response.pingerIntervalId;
```

Comment:
These are no longer optional as startModelWatcher throws in an error instead of returning null if there is an error.